tools/DocsLinter/ThirdPartyLicenses.md

```diff
@@ -0,0 +1,25 @@
+License for [Markdig](https://github.com/lunet-io/markdig): 
```

Comment:
Run past Legal

Reply:
Have done, Sophie was good with putting it into a file local to the usage. Have pinged a link to the rendered markdown to get her approval.

Reply:
Got approval, all good to go!